Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Execute_snippet_code_script.yml

```diff
@@ -1,58 +1,57 @@
-description: Initiates a new endpoint script execution action using the provided snippet
-  code and retrieves the file results.
 id: Cortex XDR - Execute snippet code script
```

Comment:
As far as I know - the developers change those commands with polling inside the command. so the playbook with polling checks isnt needed. (we can deprecate the playbook and use the command instead). 

Can you please check?